includes/class-llms-prevent-concurrent-logins.php

```diff
@@ -0,0 +1,165 @@
+<?php
+/**
+ * LLMS_Prevent_Concurrent_Logins class file
+ *
+ * @package LifterLMS/Classes
+ *
+ * @since [version]
+ * @version [version]
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * LLMS_Prevent_Concurrent_Logins class.
+ *
+ * @since [version]
+ */
+class LLMS_Prevent_Concurrent_Logins {
+
+	use LLMS_Trait_Singleton;
+
+	/**
+	 * Array of sessions for the current user.
+	 *
+	 * @var array
+	 */
+	private $user_sessions;
+
+	/**
+	 * Current user ID.
+	 *
+	 * @var int
+	 */
+	private $user_id;
+
+	/**
+	 * Private Constructor.
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	private function __construct() {
+
+		if ( get_option( 'lifterlms_prevent_concurrent_logins', false ) &&
```

Comment:
The option stores yes/no values which are handled by `llms_parse_bool()` to true/false.
```suggestion
		if ( llms_parse_bool( get_option( 'lifterlms_prevent_concurrent_logins', 'no' ) ) &&
```

Reply:
Yeah stupid me